app/Repositories/LocationRepository.php

```diff
@@ -137,12 +137,12 @@ public function analystic(
                 'title' => $title . ' ' . $count,
                 'className' => $className,
                 'url' => route(
-                    'calendar.location.detail_location',
-                    [
-                        'id' => $location->id,
-                        'date' => $date,
-                    ]
-                ) . '?shift=' . $filter['shift'],
+                        'calendar.location.detail_location',
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionCallSignature.Indent</br>**Severity**: error</br>**File**: app/Repositories/LocationRepository.php</br>**Line**: 140</br>Multi-line function call not indented correctly; expected 20 spaces but found 24